docs/general/guides/communications/communications.md

```diff
@@ -0,0 +1,114 @@
+# Communications
```

Comment:
Please change this line from "Communications" to "Messaging to voters guide", and also add something like the following, to ensure ordering in the left sidebar is always preserved (see other guides for reference):

```yaml
---
sidebar_position: 6
---
```